Assets/Scripts/UI/DialogBox/Settings/DialogBoxSettings.cs

```diff
@@ -78,7 +78,7 @@ public void OnClickSave()
         this.CloseDialog();
         WorldController.Instance.spawnInventoryController.SetUIVisibility(developerModeToggle.isOn);
         LocalizationTable.SetLocalization(languageDropdown.value);
-        //FurnitureBuildMenu.instance.RebuildMenuButtons(developerModeToggle.isOn);
+        ///FurnitureBuildMenu.instance.RebuildMenuButtons(developerModeToggle.isOn);
```

Comment:
StyleCop is fine with the `///` comments. It has issues with `//`. I changed it anyway and added a TODO